sql/src/SqlCsv.sk

```diff
@@ -885,14 +890,15 @@ fun replayDiff(
       | current @ Some(DiffParsingState(_, currentDiff)) if (
         line == "!rebuild"
       ) ->
+        SKDBMonitor.addReplicationReads(1);
```

Comment:
Worth capturing rebuilds as a separate metric. Obviously we want to minimise these.